src/measure_theory/integral/interval_integral.lean

```diff
@@ -383,25 +383,28 @@ lemma sum (s : finset ι) {f : ι → ℝ → E} (h : ∀ i ∈ s, interval_inte
   interval_integrable (∑ i in s, f i) μ a b :=
 ⟨integrable_finset_sum' s (λ i hi, (h i hi).1), integrable_finset_sum' s (λ i hi, (h i hi).2)⟩
 
-lemma mul_continuous_on {f g : ℝ → ℝ}
+lemma mul_continuous_on {A : Type*} [normed_ring A] {f g : ℝ → A}
```

Comment:
Now that you have `A` at the top, you don't need it here
```suggestion
lemma mul_continuous_on [normed_ring A] {f g : ℝ → A}
```

Ideally the `[normed_ring A] ` would move to line 366 too